examples/canoniCAI/data/ner_train.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
The indentation of this file is different from rest of the json files.

Reply:
This is auto generated from the ner_gen file because it is quite difficult to train ner manually.